msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase1561080.java

```diff
@@ -0,0 +1,208 @@
+import com.microsoft.identity.client.ui.automation.browser.BrowserEdge;
+import com.microsoft.identity.client.ui.automation.interaction.OnInteractionRequired;
+import com.microsoft.identity.client.ui.automation.interaction.PromptParameter;
+import com.microsoft.identity.client.ui.automation.interaction.microsoftsts.AadLoginComponentHandler;
+import com.microsoft.identity.client.ui.automation.interaction.microsoftsts.MicrosoftStsPromptHandler;
+import com.microsoft.identity.client.ui.automation.interaction.microsoftsts.MicrosoftStsPromptHandlerParameters;
+import com.microsoft.identity.common.java.providers.oauth2.IDToken;
+import com.microsoft.identity.labapi.utilities.client.LabQuery;
+import com.microsoft.identity.labapi.utilities.constants.TempUserType;
+import com.microsoft.identity.client.ui.automation.utils.UiAutomatorUtils;
+import com.microsoft.identity.labapi.utilities.jwt.JWTParserFactory;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.concurrent.TimeUnit;
+
+// [WPJ] Get device state
+// https://identitydivision.visualstudio.com/Engineering/_testPlans/define?planId=1905195&suiteId=1905204
+@SupportedBrokers(brokers = {BrokerMicrosoftAuthenticator.class})
+public class TestCase1561080 extends AbstractMsalBrokerTest {
+
+    final String MY_ACCOUNT_MICROSOFT_URL = "https://myaccount.microsoft.com/";
+
+    @Test
+    public void test_1561080() throws Throwable {
+        final String username = mLabAccount.getUsername();
+        final String password = mLabAccount.getPassword();
+
+        final BrokerMicrosoftAuthenticator brokerMicrosoftAuthenticator = (BrokerMicrosoftAuthenticator) mBroker;
+
+        //perform device registration
+        brokerMicrosoftAuthenticator.performDeviceRegistration(username, password);
+
+        BrowserChrome chrome  = new BrowserChrome();
+        chrome.clear();
+        chrome.launch();
+        chrome.handleFirstRun();
+
+        chrome.navigateTo(MY_ACCOUNT_MICROSOFT_URL);
+
+        final AadLoginComponentHandler aadLoginComponentHandler = new AadLoginComponentHandler();
+        aadLoginComponentHandler.handleEmailField(username);
+        aadLoginComponentHandler.handlePasswordField(password);
+
+        UiAutomatorUtils.handleButtonClick("com.android.chrome:id/button_secondary");
+
+        UiAutomatorUtils.handleButtonClickForObjectWithText("MANAGE DEVICES");
+
+        UiAutomatorUtils.handleButtonClickForObjectWithText("Collapsible item");
+
+        UiObject disable_lost_device_btn1 = UiAutomatorUtils.obtainUiObjectWithText("Disable lost device");
+        disable_lost_device_btn1.click();
+
+        UiObject disable_lost_device_btn2 = UiAutomatorUtils.obtainUiObjectWithText("Disable lost device");
+        disable_lost_device_btn2.click();
+
+        //install prod brokerhost app.
+        BrokerHost brokerHost = new BrokerHost(BrokerHost.BROKER_HOST_APK_RC);
+        if(brokerHost.isInstalled()){
```

Comment:
do we expect BrokerHost to be installed? The test case expects Authenticator to be the Broker.

Reply:
It shouldn't be installed. I think it is good to check if it was installed (by accidental automation code change) and in that case I suggest that we fail the test case as opposed to uninstalling and proceeding